settings.gradle

```diff
@@ -64,7 +64,7 @@ if (localProperties != null) {
         // As above, hacks to execute .py files on Windows.
         def publishAcCmd = [];
         if (System.properties['os.name'].toLowerCase().contains('windows')) {
-            publishAcCmd << "py";
+            publishAcCmd << "python3";
```

Comment:
What's this for? Can we move to a separate PR if it's not relevant to the padding changes?

Reply:
Same here 😅